### DISCUSSION TITLE
How to perform Single Logout in SAML2P?

### DISCUSSION THREAD

**Discussion by Wavesolid** [Aguafrommars/TheIdServer#1086] [Q&A] [answered]
I'm using sample client apps ItFoxTec, to perform logout. 

My expected result is:
1. User click login
2. user input their credentials (email & password)
3. credentials validated and authenticated
4. I press logout
5. User re-enter credentials again to login

but currently when I perform logout, user is not re-input their credentials, instead they just simply redirected back again to my client apps and authenticated as the previous user. Is there any way to delete user session in both IdP and SP? Thank you.

**Comment by Wavesolid** [comment, marked as answer]
I think I found the answer, it seems you missed to DeleteSession in logout Request, so whenever user hit Logout, the session in IdP is not really cleaned. Can I create a pull request in your repository, to fix this issue?

**Comment by aguacongas** [comment]
Hi, thx to contribute. Sure you can any help are welcomed.
